build/common/installer/scripts/tomlparser-agent-config.rb

```diff
@@ -309,6 +313,28 @@ def populateSettingValuesFromConfigMap(parsedConfig)
         @multiline_enabled = multiline_config[:enabled]
         puts "Using config map value: AZMON_MULTILINE_ENABLED = #{@multiline_enabled}"
       end
+
+      network_listener_waittime_config = parsedConfig[:agent_settings][:network_listener_waittime]
+      if !network_listener_waittime_config.nil?
+        waittime = network_listener_waittime_config[:tcp_port_25226]
+        if is_valid_number?(waittime)
```

Comment:
We should have check on lower and upper bounds . For example, lower bound is half of the default and upper bound is ~ 3 to 4x . If the value outside lower and upper bounds, we can use the default.